.github/actions/common-setup-ec2-linux/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Setup EC2 Linux
```

Comment:
Is this not already covered by `.github/actions/setup-linux`?

Reply:
This composite action calls the `.github/actions/setup-linux` action and does some extra work before & after it